README.md

```diff
@@ -47,6 +49,18 @@ export default function App() {
 }
 ```
 
+## Usage with Immer
+
+Somewhere globally like index.ts or App.ts
```

Comment:
Can it be scoped to single file as well, or is it purely global?

Reply:
Well, we could do an optional third argument to useMicroReducer?

Reply:
Maybe that's actually better. You might have to include immer more places, but on the other hand WYSIWYG and it's more clear that the state has been modified

Reply:
so it'd be either
```
// app.tsx
setImmer(produce);

// file.tsx
const [...] = useMicroReducer(actions, state);
```
or 
```
// file.tsx
const [...] = useMicroReducer(actions, state, producer);
```

Reply:
That's actually pretty cool. Then what if you already have a global producer?

Reply:
Local always wins

Reply:
But I'm thinking to not do the global version at all. Is it making more confusion? Imagine opening a file and seeing the state getting mutated, at least by explicitly passing it you can always see that something is modifying it.

Reply:
True. Makes good sense :)

Reply:
Check the code now. How do you like them Apples? :D